mp/game/momentum/resource/SourceScheme.res

```diff
@@ -149,7 +151,7 @@ Scheme
 
         ListPanel.TextColor					"White"
         ListPanel.TextBgColor				"Blank"
-        ListPanel.BgColor					"MomGreydientStep4"
+        // ListPanel.BgColor					"MomGreydientStep4"
```

Comment:
Was this meant to be removed instead of commented out? Not too sure as other things in this file are commented out.

Reply:
Was commented out as the next couple lines has the new value, and forgot to remove this one. Removed it.